### PR TITLE
Fixing Reaction Chambers Part 4 Revengeance

### PR DESCRIPTION
**Pull request by Pugie**
## About The Pull Request
Pleaseworkthistimepleaseworkthistimepleaseworkthistimeplease
Re-adds the check for if reaction chamber will give reagents to ductnets, that was removed in fermichem removal.
Fixes https://github.com/AetherStation/AetherStation13/issues/131

## Why It's Good For The Game
image

## Changelog
:cl:
fix: Chems will now stop to react in reaction chamber.
/:cl:

### CODE REVIEW

**Comment by TheChosenEvilOne** [comment]
what should we call the fifth one?

File: code/datums/components/plumbing/reaction_chamber.dm [hunk 1]
```diff
@@ -7,6 +7,12 @@
 	if(!istype(parent, /obj/machinery/plumbing/reaction_chamber))
 		return COMPONENT_INCOMPATIBLE
 
+/datum/component/plumbing/reaction_chamber/can_give(amount, reagent, datum/ductnet/net)
+	. = ..()
+	var/obj/machinery/plumbing/reaction_chamber/reaction_chamber = parent
+	if(!reaction_chamber.emptying)
+		return FALSE
```
Comment (TheChosenEvilOne) [inline code/datums/components/plumbing/reaction_chamber.dm:14]: this could just be
```suggestion
	return (. && reaction_chamber.emptying)
```

Comment (Pugie) [reply code/datums/components/plumbing/reaction_chamber.dm:14]: ok

**Comment by Pugie** [comment]
If this breaks again I'm going to be sent into a coma

**Comment by TheChosenEvilOne** [comment]
> If this breaks again I'm going to be sent into a coma

this is all for the fifth PR

**Comment by Sparkezel** [comment]
chemistry device liquid 5

**Review by TheChosenEvilOne — APPROVED** [review]

**Pull request merged**